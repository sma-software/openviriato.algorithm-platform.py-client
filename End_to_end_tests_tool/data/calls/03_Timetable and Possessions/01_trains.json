{
    "restResource": "/trains",
    "httpMethod": "GET",
    "urlParameters": null,
    "queryParameters": [
        {
            "key": "FromTime",
            "value": "2003-05-01T00%3A05%3A00",
            "optional": false
        },
        {
            "key": "ToTime",
            "value": "2003-05-01T00%3A10%3A00",
            "optional": false
        }
    ],
    "requestBody": null,
    "CSharpMethodName": "GetTrains(timeWindow)",
    "CSharpMethodParameterTypes": [ "SMA.AlgorithmPlatform.AlgorithmInterface.AIDM.ITimeWindow" ],
    "subcategory" : "Get Trains" 
}
