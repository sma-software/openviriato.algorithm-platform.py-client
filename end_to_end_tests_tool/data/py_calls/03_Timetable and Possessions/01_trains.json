{
  "PythonParameterFromJson" :
  [
      {
          "ParameterName" : "time_window",
          "aidm_factory": "convert_json_to_time_window",
          "ObjectJson":
          {
              "from_time": "2003-05-01T00:05:00",
              "to_time": "2003-05-01T00:10:00"
          }
      }
  ]
}