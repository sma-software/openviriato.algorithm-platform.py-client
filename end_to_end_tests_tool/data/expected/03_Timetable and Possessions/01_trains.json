[
  {
    "id": 3516,
    "trainPathNodes": [
      {
        "id": 2424,
        "sectionTrackId": null,
        "nodeId": 7,
        "nodeTrackId": 8,
        "formationId": 1187,
        "arrivalTime": "2003-05-01T00:10:00",
        "departureTime": "2003-05-01T00:10:00",
        "minimumRunTime": null,
        "minimumStopTime": "P0D",
        "stopStatus": "commercialStop",
        "sequenceNumber": 0
      },
      {
        "id": 2788,
        "sectionTrackId": 1175,
        "nodeId": 15,
        "nodeTrackId": 16,
        "formationId": 1187,
        "arrivalTime": "2003-05-01T00:14:00",
        "departureTime": "2003-05-01T00:14:00",
        "minimumRunTime": "PT4M",
        "minimumStopTime": "P0D",
        "stopStatus": "passing",
        "sequenceNumber": 1
      },
      {
        "id": 3152,
        "sectionTrackId": 1174,
        "nodeId": 10,
        "nodeTrackId": 12,
        "formationId": null,
        "arrivalTime": "2003-05-01T00:19:00",
        "departureTime": "2003-05-01T00:19:00",
        "minimumRunTime": "PT5M",
        "minimumStopTime": "P0D",
        "stopStatus": "commercialStop",
        "sequenceNumber": 2
      }
    ],
    "debugString": "RVZH_3_1_J03 tt_(S)"
  },
  {
    "id": 2060,
    "trainPathNodes": [
      {
        "id": 1332,
        "sectionTrackId": null,
        "nodeId": 18,
        "nodeTrackId": null,
        "formationId": 1187,
        "arrivalTime": "2003-05-01T00:05:00",
        "departureTime": "2003-05-01T00:05:00",
        "minimumRunTime": null,
        "minimumStopTime": "P0D",
        "stopStatus": "commercialStop",
        "sequenceNumber": 0
      },
      {
        "id": 1696,
        "sectionTrackId": 1171,
        "nodeId": 10,
        "nodeTrackId": null,
        "formationId": null,
        "arrivalTime": "2003-05-01T00:10:00",
        "departureTime": "2003-05-01T00:10:00",
        "minimumRunTime": "PT5M",
        "minimumStopTime": "P0D",
        "stopStatus": "commercialStop",
        "sequenceNumber": 1
      }
    ],
    "debugString": "RVZH_1_1_J03 tt_(G)"
  }
]